cmd/podman/machine/cp.go

```diff
@@ -0,0 +1,144 @@
+//go:build amd64 || arm64
+
+package machine
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	"github.com/containers/podman/v5/libpod/events"
+	"github.com/containers/podman/v5/pkg/copy"
+	"github.com/containers/podman/v5/pkg/machine/define"
+	"github.com/containers/podman/v5/pkg/machine/env"
+	"github.com/containers/podman/v5/pkg/machine/vmconfigs"
+	"github.com/spf13/cobra"
+)
+
+type cpOptions struct {
+	Quiet    bool
+	Machine  *vmconfigs.MachineConfig
+	IsSrc    bool
+	SrcPath  string
+	DestPath string
+}
+
+var (
+	cpCmd = &cobra.Command{
+		Use:               "cp [options] src_path dest_path",
```

Comment:
SRC_PATH and DEST_PATH must be upper case. I am not sure how constent the cli si but all the command arguments must be in upper case.
I think we use lower case in the man page though.